components/match2/commons/starcraft_starcraft2/match_summary_starcraft.lua

```diff
@@ -24,7 +25,7 @@ local Opponent = OpponentLibraries.Opponent
 local OpponentDisplay = OpponentLibraries.OpponentDisplay
 
 local ICONS = {
-	greenCheck = '<i class="fa fa-check forest-green-text" style="width: 14px; text-align: center" ></i>',
+	greenCheck = Icon.makeIcon{iconName = 'check', color = 'forest-green-text', size = 'initial'},
```

Comment:
'initial' intended?

Reply:
yes, per @salle

Reply:
it changes the display:

left = dev;; right = live
![Screenshot 2024-01-26 132619](https://github.com/Liquipedia/Lua-Modules/assets/75081997/0a8da8eb-7642-4a16-b6e1-094c84032891)

in dev (with `size = 'initial'`) the checkmark is bigger

not sure that is wanted (personally i like the smaller ones a bit better but can live with the bigger ones too)

Reply:
The best option would have been to set it  as a percentage based font size, then whatever change we do to the fonts or font sizes in the future this will just go along with, so we will save more time if we need to update the site. Using Initial is just slightly better than setting the font size in pixels but it requires much less time than setting up what % it should be. 

So if you can figure out what the % font-size should be for the icons here to make them 14 pixels wide I am all for using that instead. They are 11.89 pixels without a font size change so I think the % needs to be 117.8% but I didn't try it out, and it might need to be 87.5% if it's calculated from the 16px which is the default site size.

Reply:
so for sc/sc2:
`initial` sets ot to 16
before it was 14
100% is 13
so 14 would be ~107.7%
(also checked with dev tools on sc2)

have to double check on other wikis (since sc(2) uses some sc(2) specific classes for match2)
but should be the same though iirc

Reply:
other wikis the same

maybe go to 108% to not have fractions in the %?

Reply:
It should be fine to use fractions, as far as I'm aware. If you want to round the number the common one would be 110%

Reply:
110% is fine for me

Reply:
Changed it all to 110%. It's gonna have slightly different effects on  different wikis. Some use fixed size for game rows (eg. SC2), some use 85% and some use 80%. Let's go with 110% for all and adjust afterwards if needed on individual wikis.